crates/geom/src/arc.rs

```diff
@@ -1049,20 +1049,4 @@ fn negative_flattening_step() {
     };
 
     arc.for_each_flattened(0.100000001, &mut |_| {});
-
-    // There was also an issue with negative sweep_angle leading to a negative step
```

Comment:
It looks like you removed this part of the test. Was it a rebasing error?

Reply:
Yep, I've added it back.